### PR TITLE
Switch to build rs

### PR DESCRIPTION
**Pull request by andrewzah**

### CODE REVIEW

**Comment by andrewzah** [comment]
#21

File: build.rs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    let mut protos_cmd = Command::new("protoc")
+        .arg("--encode=Maeve.Game")
+        .arg("protos/master.proto")
+        .stdin(Stdio::piped())
+        .stdout(Stdio::from(output_file))
+        .spawn()
+        .expect("Failed to compile pb file.");
+
+    /* Assign _ because it must be used */
+    let _ = protos_cmd
```
Comment (dmadisetti) [inline build.rs:44]: Can you do ".expect()"?

**Review by dmadisetti — COMMENTED** [review]
LGTM. Can you update the readme while you're at it?

File: build.rs (hunk 1 above)
Comment (andrewzah) [reply build.rs:44]: Added an [expect()](https://github.com/AnotherGroupChat/maeve/pull/36/commits/e544f582155b255da0acaa441a2515410ad21c3e#diff-a7b0a2dee0126cddf994326e705a91eaR49)

Will update the readme tomorrow

**Comment by kevinlmadison** [comment]
Looks good, can we merge and i'll change the readme?

**Pull request merged**